CHANGELOG.md

```diff
@@ -0,0 +1,82 @@
+<!--
+How to Maintain This Changelog
+
+### For Contributors
+- Add entries to the "Unreleased" section when making changes.
+- Use the appropriate category (Added, Changed, Fixed, etc.).
+- Write entries from the user's perspective.
+- Reference issue numbers when applicable.
+
+### For Maintainers
+- Move "Unreleased" items to a new version section when releasing.
+- Add the release date.
+- Update the comparison links at the bottom.
+- Tag the release in Git with the version number.
+
+### Categories
+- **Added** for new features.
+- **Changed** for changes in existing functionality.
+- **Deprecated** for soon-to-be removed features.
+- **Removed** for now removed features.
+- **Fixed** for any bug fixes.
+- **Security** in case of vulnerabilities.
+-->
+
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+### Added
+- _Nothing yet_
+
+### Changed
+- _Nothing yet_
+
+### Deprecated
+- _Nothing yet_
+
+### Removed
+- _Nothing yet_
+
+### Fixed
+- _Nothing yet_
+
+### Security
+- _Nothing yet_
+
```

Comment:
I don't know we want this visible either. What do you think about moving it to the commented out point?